build.xml

```diff
@@ -1437,7 +1609,7 @@
           <include name="**/*.jar" />
         </fileset>
       </classpath>
-      <jvmarg line="${java11-jvmargs}"/>
+      <jvmarg line="${java-jvmargs}"/>
```

Comment:
Good catch! I don't think we run that target in CI. Maybe we should explore to add it?